toqito/measurements/pretty_bad_measurement.py

```diff
@@ -8,21 +8,16 @@
 def pretty_bad_measurement(
     states: list[np.ndarray], probs: list[float] | None = None, tol: float = 1e-8
 ) -> list[np.ndarray]:
-    r"""Return the set of pretty bad measurements from a set of vectors and corresponding probabilities.
+    r"""Return the set of pretty bad measurements from an ensemble.
```

Comment:
I think either line is technically correct here. Some users might not know that an "ensemble" is just a set of vectors and a set of probabilities. Additionally, the function arguments are separate--that is, we don't accept an "ensemble", but we accept these two items separately, so we should specify them separately as well.